funsor/torch/__init__.py

```diff
@@ -4,13 +4,13 @@
 import torch
 from multipledispatch import dispatch
 
+import funsor.ops as ops
```

Comment:
I believe you can resolve the Travis error by adding a line below like `from funsor.ops import AssociativeOp` and changing `ops.AssociativeOp` to `AssociativeOp`.